ouroboros-network/protocol-tests/Ouroboros/Network/Protocol/Handshake/Test.hs

```diff
@@ -539,6 +545,103 @@ prop_pipe_asymmetric_IO (ArbitraryVersions clientVersions _serverVersions) =
     ioProperty (prop_channel_asymmetric createPipeConnectedChannels clientVersions)
 
 
+{--
+  - prop_connect_clients
+  -     :: ArbitraryVersions
+  -     -> Property
+  - prop_connect_clients (ArbitraryVersions clientVersions clientVersions') =
+  -   let (eClientRes, eClientRes') =
+  -         pureHandshake
+  -           ((maybeAccept .) . acceptableVersion)
+  -           clientVersions'
+  -           clientVersions
+  -   in case runSimOrThrow
+  -            (connect_clients
+  -               (handshakeClientPeer
+  -                 (cborTermVersionDataCodec dataCodecCBORTerm)
+  -                 acceptableVersion
+  -                 clientVersions)
+  -               (handshakeClientPeer
+  -                 (cborTermVersionDataCodec dataCodecCBORTerm)
+  -                 acceptableVersion
+  -                 clientVersions')) of
+  -       (rClientRes, rClientRes', TerminalStates TokDone TokDone) ->
+  -            fromMaybe False eClientRes
+  -            ===
+  -            either (const False) (\(a,_,_) -> a) rClientRes
+  -         .&&.
+  -            fromMaybe False eClientRes'
+  -            ===
+  -            either (const False) (\(a,_,_) -> a) rClientRes'
+  --}
```

Comment:
Why do you need to commit this commented out code?
If it is needed can you add a comment on why it is worth keeping around.

Reply:
This should be removed.